frontend/src/cards/TableCard.tsx

```diff
@@ -66,7 +66,7 @@ export function TableCard(props: TableCardProps) {
     )
   );
 
-  let metricConfigs: Record<string, MetricConfig> = {};
+  let metricConfigs: Partial<Record<MetricId, MetricConfig>> = {};
```

Comment:
Nice! I'm just learning about the Partial type property.

Reply:
Ya it's pretty slick; otherwise TypeScript Record requires you to map every possible item; the Partial lets you just define some of them